lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -286,6 +289,71 @@ class ConvertTritonGPUOpToLLVMPatternBase {
     return base;
   }
 
+  // Generate a pretty string for basic pointer calculation via GEP and binary
+  // operations.
+  std::string printPtrExpr(Value val) const {
```

Comment:
This function is to provide an  easier-to-read dump of the pointer arithmetic. I am not certain if this is useful or reliable enough.